backend/tests/db/test_worker.py

```diff
@@ -2,30 +2,34 @@
 
 from sqlalchemy.orm import Session as OrmSession
 
-from mirrors_qa_backend.db import models, worker
+from mirrors_qa_backend.db.models import Country
+from mirrors_qa_backend.db.worker import create_worker
 
 
-def test_create_worker(dbsession: OrmSession, tmp_path: Path):
+def test_create_worker(dbsession: OrmSession, tmp_path: Path, private_key_bytes: bytes):
```

Comment:
```suggestion
def test_create_worker(dbsession: OrmSession, tmp_path: Path, private_key: bytes):
```

We already have type hint and there's no risk of confusion. Use `private_key_data` or `private_key_content` if you prefer